README.md

```diff
@@ -181,6 +181,17 @@ Then, specify all the required certificates that need to be completed before usi
 ```
 Default is an empty list.
 
+### File mapping configuration
```

Comment:
these should go into the template of this PR and probably in a seperated doc under `/docs`

probably also an good idea to update https://github.com/uc-cdis/cloud-automation/blob/master/kube/services/portal/README.md as well